lib/roby/actions/models/interface_base.rb

```diff
@@ -73,18 +73,35 @@ def clear_model
             # @return Action
             def describe(doc = nil)
                 if @current_description
-                    Actions::Interface.warn "the last #describe call was not followed by an action definition. Did you forget to add a method to your action interface ?"
+                    Actions::Interface.warn "the last #describe call was not followed by "\
+                    "an action definition. Did you forget to add a method to your action interface ?"
                 end
                 @current_description = Models::Action.new(doc)
             end
 
             # Registers a new action on this model
+            # This function is meant to be used to share actions across action interfaces
```

Comment:
- empty line between first and second line (for documentation generators)
- give an example
- move the next paragraph, to `register_action!`, where it is more relevant.